objects/Seed.go

```diff
@@ -212,10 +213,12 @@ func GetManifestLabel(seedFileName string) string {
 			err.Error())
 	}
 
+	util.PrintUtil("GetManifestLabel.seedbytes:\n%s\n", string(seedbytes))
+
 	// Escape forward slashes and dollar signs
 	seed := string(seedbytes)
-	seed = strings.Replace(seed, "$", "\\$", -1)
-	seed = strings.Replace(seed, "/", "\\/", -1)
+	// seed = strings.Replace(seed, "$", "\\$", -1)
+	// seed = strings.Replace(seed, "/", "\\\\/", -1)
 
```

Comment:
I'd check that nothing calling this method expects these strings to be commented out. Specifically I'm not sure if SILO expects this or not.

Reply:
Also remove commented out lines.

Reply:
Looks like I didn't update my latest before creating the PR. These line were actually necessary, so they should be uncommented. Good catch.